### PR TITLE
Update estimate calls to wallet pay with rsa enabled

### PR DESCRIPTION
**Pull request by zliu7878**
- Updated braintree google pay/ apple pay
- Updated stripe google pay/ apple pay
- Updated paypal express

### CODE REVIEW

**Review by mitchell-engen — APPROVED** [review]

File: src/braintree/apple/braintreeOnShippingContactSelectedApple.ts [hunk 1]
```diff
@@ -29,37 +34,46 @@ export async function braintreeOnShippingContactSelectedApple(event: ApplePayShi
...
-            const {totalAmountDue} = getTotals();
-            const displayItems = getPaymentRequestDisplayItems().map(
-                ({label, amount}) => ({
-                    label,
-                    amount: getValueByCurrency(amount, currencyCode),
-                }));
-            const {available_shipping_lines: shippingLines} = getShipping();
+    let responseSuccess = false;
+    if (rsaEnabled) {
+        const estimateShippingResponse = await estimateShippingLines(address, API_RETRY);
```
Comment (Shagufa92) [inline src/braintree/apple/braintreeOnShippingContactSelectedApple.ts:40]: This wont set the default shipping line to the order. This only gets the available shipping lines from the platform. Feel free to resolve the thread if this scenario is already covered.

File: src/braintree/google/braintreeOnPaymentDataChangeGoogle.ts [hunk 2]
```diff
@@ -18,13 +21,18 @@ export async function braintreeOnPaymentDataChangeGoogle(intermediatePaymentData
...
     switch (callbackTrigger) {
         case braintreeConstants.GOOGLEPAY_TRIGGER_INITIALIZE:
         case braintreeConstants.GOOGLEPAY_INTENT_SHIPPING_OPTION:
         case braintreeConstants.GOOGLEPAY_INTENT_SHIPPING_ADDRESS: {
-            if (shippingAddress) {
-                const address = formatBraintreeShippingAddressGoogle(shippingAddress, true);
+            let shippingLinesResponse;
+            const address = formatBraintreeShippingAddressGoogle(shippingAddress, true);
+            if (rsaEnabled) {
+                shippingLinesResponse = await estimateShippingLines(address, API_RETRY);
```
Comment (praiseBold) [inline src/braintree/google/braintreeOnPaymentDataChangeGoogle.ts:34]: re: @Shagufa92 comment probably applies here too

File: src/braintree/apple/braintreeOnShippingContactSelectedApple.ts (hunk 1 above)
Comment (zliu7878) [reply src/braintree/apple/braintreeOnShippingContactSelectedApple.ts:40]: Thank you for catch this, added the set default shipping line to the order.

File: src/braintree/google/braintreeOnPaymentDataChangeGoogle.ts (hunk 2 above)
Comment (zliu7878) [reply src/braintree/google/braintreeOnPaymentDataChangeGoogle.ts:34]: Thank you for catch this, fixed as well

**Review by praiseBold — APPROVED** [review]

**Pull request merged**